Sources/TokamakCore/Fiber/Fiber.swift

```diff
@@ -54,11 +54,15 @@ public extension FiberReconciler {
     /// all stored properties be set before using.
     /// `outputs` is guaranteed to be set in the initializer.
     var outputs: ViewOutputs!
+    /// The erased `Layout` to use for this content.
```

Comment:
nit: while this file is edited, could we add a newline between a previous property and a doc comment of a following property for readability? There are obviously more places in this file where this change could be applied.
```suggestion

    /// The erased `Layout` to use for this content.
```